frame/contracts/src/weights.rs

```diff
@@ -153,1251 +153,1321 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	fn on_initialize() -> Weight {
-		(3_850_000 as Weight)
+		(3_664_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 	}
 	fn on_initialize_per_trie_key(k: u32, ) -> Weight {
-		(52_925_000 as Weight)
-			// Standard Error: 5_000
-			.saturating_add((2_297_000 as Weight).saturating_mul(k as Weight))
+		(0 as Weight)
+			// Standard Error: 2_000
+			.saturating_add((2_320_000 as Weight).saturating_mul(k as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(k as Weight)))
 	}
 	fn on_initialize_per_queue_item(q: u32, ) -> Weight {
-		(434_698_000 as Weight)
-			// Standard Error: 210_000
-			.saturating_add((166_559_000 as Weight).saturating_mul(q as Weight))
+		(0 as Weight)
+			// Standard Error: 34_000
+			.saturating_add((34_529_000 as Weight).saturating_mul(q as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn instrument(c: u32, ) -> Weight {
-		(29_918_000 as Weight)
-			// Standard Error: 185_000
-			.saturating_add((123_774_000 as Weight).saturating_mul(c as Weight))
+		(0 as Weight)
+			// Standard Error: 353_000
+			.saturating_add((2_325_912_000 as Weight).saturating_mul(c as Weight))
```

Comment:
Wow. Costs to instrument the code shot up by 20x per KB since the last benchmark. Not sure why this is. This PR did not change the instrumentation code. Maybe wasm-utils was updated and contains a performance regression?